### DISCUSSION TITLE
JVMChaos experiments and Byteman

### DISCUSSION THREAD

**Discussion by jfsmith-at-coveo** [chaos-mesh/chaos-mesh#3703] [Q&A] [answered]
Greetings everyone!

This may sound like a silly question, but it is unclear to me whether or not I need to install Byteman myself on the containers that run the java application I want to target with a JVMchaos experiment.

Sorry for the naive question! Thanks in advance,
JFS

**Comment by cwen0** [comment, marked as answer]
@jfsmith-at-coveo You don't need to install Byteman on your containers manually.  Chaos Mesh will attach Byteman to your containers by chaos-daemon automatically.

**Reply by jfsmith-at-coveo** [reply]
That's fabulous! Thank you for the quick answer.
